iceoryx_hoofs/include/iceoryx_hoofs/cxx/expected.hpp

```diff
@@ -160,6 +160,7 @@ class IOX_NO_DISCARD expected<ErrorType> : public FunctionalInterface<expected<E
     /// ErrorType to correctly invalidate the stored object
     expected(expected&& rhs) noexcept;
 
+    // AXIVION DISABLE STYLE AutosarC++19_03-A16.0.1: macro is used to reduce boilerplate code for 'NewType'
 #if defined(_WIN32)
```

Comment:
Did you already try to remove/disable this and check if it is still required by windows? You can easily try this by changing `#if defined(_WIN32)` to `#if 0` and check if the CI on Windows runs successfully. If this is the case, the whole code could be removed.

Reply:
@elBoberido I have tried the same and observed the CI on Windows fails.